sections/deckbuilding.tex

```diff
@@ -21,18 +21,20 @@ \subsection*{General Card Rules}
     Also, whenever one of these Discard Piles is empty, \textbf{refill it} with that Deck's top Card.
   \item Cards in Your Deck have the following types of effects:
   \begin{itemize}
-    \item \textbf{Instant} \includesvg[height=10px]{\svgs/instant.svg} Effects are resolved immediately and can be played \textbf{out of Turn during Combat}.
+    \item \textbf{Instant} \includesvg[height=10px]{\svgs/instant.svg} Effects are resolved immediately.
     \item \textbf{Activation} \includesvg[height=10px]{\svgs/activation.svg} Effects must be played when Activating your own Unit in Combat.
     \item \textbf{Map} \includesvg[height=10px]{\svgs/map.svg} Effects cannot be used during Combat.
     \item \textbf{Ongoing} \includesvg[height=10px]{\svgs/ongoing.svg} Effects last until they are used up or until the player who played them starts their next Turn (whichever happens first).
-    \item \textbf{Permanent} \includesvg[height=10px]{\svgs/permanent.svg} Cards\index{Permanent Cards} stay in your play area until discarded or replaced.
-      \textbf{Players may have only one permanent Card at a time}; playing another discards the first.
+    \item \textbf{Permanent} \includesvg[height=10px]{\svgs/permanent.svg} Cards\index{Permanent Cards} stay in play until discarded or replaced.
+    \textbf{You may have only one permanent Card at a time}; playing another discards the first.
```

Comment:
```suggestion
      \textbf{You may have only one permanent Card at a time}; playing another discards the first.
```
Leave this indenation, as it relates to the previous bullet point. It's not a standalone one.